Initial space refactor attempt (#384)
upbound/docs: content/disconnected-spaces/_index.md

---
title: Disconnected Spaces
weight: 2
description: A guide to Upbound Spaces
---

Upbound Spaces are self-hosted environments for Upbound's managed Crossplane control planes. All that's required to bootstrap a Space is a Kubernetes cluster. Popular managed Kubernetes services including Amazon EKS, Google GKE, and Microsoft AKS are fully supported.

## Spaces and Upbound

> Think of an Upbound Space as a self-managed slice of the Upbound platform in your own environment. Whether that's a hyper scale cloud provider or on-prem, Spaces supplements Upbound's SaaS service by enabling a new deployment option. Now you can have control planes that run in your preferred hosting environment and Upbound's own SaaS.

Reviewer: ```suggestion
An Upbound Space is a self-managed instance of the Upbound platform in your own environment. Spaces adds value to Upbound's SaaS service by enabling you to choose where you run your control planes. Now, you can deploy control planes that run in your preferred hosting environment and Upbound's own SaaS.

```

{{<img src="/spaces/images/mcps-everywhere.png" alt="Managed control planes can run anywhere, thanks to Spaces">}}

Spaces lets you:

- Increase scale and cost efficiency by running 50 or more control planes per Kubernetes Cluster instead of just 1.

> - Each control plane configuration is fully managed from Git.

Reviewer: ```suggestion
- Manage each control plane configuration in source control.
```

- Integrate with Kubernetes ecosystem tooling to manage the full lifecycle of the control planes.

> The boundary of a single Upbound Space is within the confines of a single Kubernetes cluster. Spaces are regional hosting environments within a single cloud provider. You can deploy multiple Spaces (as long as each Space deploys in its own cluster). Upbound recommends deploying a Space for each region and cloud provider you want to operate managed control planes in.

Reviewer: ```suggestion
Each Upbound Space is confined within a single Kubernetes cluster. Spaces are regional hosting environments within a single cloud provider. You can deploy multiple Spaces (as long as each Space deploys in its own cluster). Upbound recommends deploying seperate Spaces for each region and cloud provider you want to use managed control planes.
```
